website/docs/developer-docs/setup/debugging.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Debugging authentik
+---
+
+This page describes how to debug different components of an authentik instance, running either in production or in a development setup. To learn more about the structure of authentik, refer to our [architecture documentation](../../core/architecture.).
+
+## authentik Server & Worker (Python)
+
+The majority of the authentik codebase is in Python, running in Gunicorn for the server and Celery for the worker. These instructions show how this code can be debugged/inspected.
```

Comment:
Add note that for local debugging, and environment setup like described in https://deploy-preview-12809--authentik-docs.netlify.app/docs/developer-docs/setup/full-dev-environment setup